.github/workflows/sdk-insertion-bump.yml

```diff
@@ -0,0 +1,34 @@
+name: Notify release branch change
+
+on:
+  # trigger for main and release branches.
+  push:
+    branches:
+      - '*'
```

Comment:
@jonpryor @pjcollins are GitHub Actions enabled on this repo? I was thinking we should see this new job run on the latest commit here:

https://github.com/xamarin/xamarin-android/tree/dev/mandel/sdk-insertions-bump

Just to know if this action will work or not.

Reply:
AFAIK it won't. I updated the commit message with a pointer to an example in the macios repo.

Reply:
I think we will want to change this branch filter to include `main` and `release/*` instead of a full wildcard if possible.

I am not sure if we need to do anything special to enable actions or to get this to run, it's probably ok to merge this and test it live.